python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -40,8 +40,9 @@
     get_calendar,
 )
 
-# Accelerated pandas has the real pandas module as an attribute
+# Accelerated pandas has the real pandas and cudf modules as an attributes
 pd = xpd._fsproxy_slow
+cudf = xpd._fsproxy_fast
```

Comment:
What happens if you `import cudf` rather than accessing it through this proxy? Does that fix the issues with monkeypatching?

Reply:
No, they still occur. I was originally importing cudf before I switched to the proxy.